scripts/vendor

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+set -eu
+
+TYP=$1
+
+usage() {
+  echo "usage: ./scripts/vendor <update|outdated>"
+  exit 1
+}
+
+if [ -z "$TYP" ]; then
+  usage
+fi
+
+# create dummy go.mod, see comment in vendor.mod
+cat > go.mod <<EOL
+module github.com/docker/cli
+
+go 1.16
+EOL
+
+update() {
+  go mod tidy -modfile=vendor.mod
```

Comment:
I've not add `go mod tidy` in https://github.com/docker/swarmkit/pull/3046 and https://github.com/moby/moby/pull/43075. Personally, I'd prefer to run it explicitly.

Reply:
I think go.mod tidiness is important to avoid dependencies drifts in our vendor folder. Maybe we should only add that in the validation stage though.

Reply:
Good point, I'll amend both of my PR to do the same :)